resources/linux/code.appdata.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
```

Comment:
Sure, I'll push an update later without the `.appdata.xml`, and leave the commit in a local branch waiting to make a follow-up PR later on.